app/controllers/landing_controller.rb

```diff
@@ -0,0 +1,4 @@
+class LandingController < ApplicationController
```

Comment:
Why dont you extend this from ActionController::Base and leave everything like it was before?